neuromation/api/__init__.py

```diff
@@ -165,10 +165,24 @@ async def login_headless(
     *,
     url: URL = DEFAULT_API_URL,
     path: Optional[Path] = None,
-    timeout: aiohttp.ClientTimeout = DEFAULT_TIMEOUT
+    timeout: aiohttp.ClientTimeout = DEFAULT_TIMEOUT,
 ) -> None:
     await Factory(path).login_headless(get_auth_code_cb, url=url, timeout=timeout)
 
 
 async def logout(*, path: Optional[Path] = None) -> None:
     await Factory(path).logout()
+
+
+class NoProjectRoot(Exception):
+    pass
+
+
+def find_project_root() -> Path:
+    here = Path.cwd()
```

Comment:
Please consider optional passing a directory as a starting point. The function signature can look like `def find_project_root(cwd: Optional[Path] = None) -> Path: ...`